nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -3123,4 +3123,32 @@ public void springAutowiredConstructorTest() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void multipleNullableParamPassTest() {
+    defaultCompilationHelper
```

Comment:
Since these tests don't really check for the desired behavior, I think we can just delete them.  I don't think the Error Prone tests allow for checking for multiple errors on the same line.  @nimakarimipour confirmed to me that he tested outside Error Prone and saw the desired output